drivers/clock_control/Kconfig.npcx

```diff
@@ -0,0 +1,43 @@
+# NPCX Clock controller driver configuration options
+
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config CLOCK_CONTROL_NPCX
+	bool "NPCX clock controller driver"
+	depends on SOC_FAMILY_NPCX
+	help
+	  Enable support for NPCX clock controller driver.
+	  
+config CLOCK_NPCX_OSC_CYCLES_PER_SEC
+	int "CDCG PLL frequency"
+	default 48000000
+	range 10000000 100000000
+	depends on SOC_FAMILY_NPCX
+	help
+	  CDCG PLL frequency, allowed values: From 10Mhz to 100Mhz.
```

Comment:
Can you write out abbreviations in full in Kconfig help? It is useful for people new to it who don't know anything about it.

Reply:
I think writing is out would be helpful too.

Also is PLL frequency something that we would want in the device tree definition instead of KConfig? (@carlescufi @nashif); Can question for all of the prescalars below

Reply:
Sure. I will fix it.